apps/src/applab/designElements/textarea.jsx

```diff
@@ -231,7 +231,7 @@ export default {
   onPropertyChange: function (element, name, value) {
     switch (name) {
       case 'value':
```

Comment:
Could we add a comment here to explain why we use the property `value` even though users don't really use `value`?